src/tests/ports/nfts-rentalsNFTComponent.spec.ts

```diff
@@ -0,0 +1,357 @@
+      let rentalAssetTokenId: string
+      let rentalAsset: RentalAsset
+
+      beforeEach(() => {
+        rentalAssetTokenId = 'tokenId'
+        rentalAsset = {
+          id: `${mockContractAddresses.land}-${rentalAssetTokenId}`,
+          contractAddress: mockContractAddresses.land,
+          tokenId: rentalAssetTokenId,
+          isClaimed: false,
+          lessor: 'owner',
+        }
+
+        mockRentalsComponent.getRentalAssets = jest
+          .fn()
+          .mockResolvedValue([rentalAsset])
+      })
+
+      it('should call fetch with ids based on the obtained rental asset that is a land', async () => {
+        await rentalsNFTComponent.count({})
+
+        expect(mockMarketplaceNFTsComponent.count).toHaveBeenCalledWith(
+          expect.objectContaining({
+            ids: [`parcel-${rentalAsset.id}`],
+          })
+        )
+      })
+
+      it('should call fetch without owner in the filters', async () => {
+        await rentalsNFTComponent.count({ owner: 'owner' })
+
+        expect(mockMarketplaceNFTsComponent.count).not.toHaveBeenCalledWith(
+          expect.objectContaining({
+            owner: 'owner',
+          })
+        )
+      })
+    })
+
+    describe('when getRentalAssets has returned one rental asset that is an estate', () => {
+      let rentalAssetTokenId: string
+      let rentalAsset: RentalAsset
+
+      beforeEach(() => {
+        rentalAssetTokenId = 'tokenId'
+        rentalAsset = {
+          id: `${mockContractAddresses.estate}-${rentalAssetTokenId}`,
+          contractAddress: mockContractAddresses.estate,
+          tokenId: rentalAssetTokenId,
+          isClaimed: false,
+          lessor: 'owner',
+        }
+
+        mockRentalsComponent.getRentalAssets = jest
+          .fn()
+          .mockResolvedValue([rentalAsset])
+      })
+
+      it('should call fetch with ids based on the obtained rental asset that is an estate', async () => {
+        await rentalsNFTComponent.count({})
+
+        expect(mockMarketplaceNFTsComponent.count).toHaveBeenCalledWith(
+          expect.objectContaining({
+            ids: [`estate-${rentalAsset.id}`],
+          })
+        )
+      })
+    })
+  })
+
+  describe('when marketplaceNFTsComponent returns a count of 1', () => {
+    beforeEach(() => {
+      mockMarketplaceNFTsComponent.count = jest.fn().mockResolvedValue(1)
+    })
+
+    it('should finally return a count of 1', async () => {
+      const result = await rentalsNFTComponent.count({})
+
+      expect(result).toBe(1)
+    })
+  })
+})
+
+describe('when calling rentalNFTComponentShouldFetch', () => {
+  let filters: NFTFilters
+
+  beforeEach(() => {
+    filters = {
+      owner: 'owner',
+      isOnRent: false,
+      isLand: true,
+    }
+  })
+
+  describe('when called without owner', () => {
+    beforeEach(() => {
+      delete filters.owner
+    })
+
+    it('should return false', () => {
+      expect(rentalNFTComponentShouldFetch(filters)).toBeFalsy()
```

Comment:
We can use falsy or truthy, but by checking for `false`, we can be sure that this is going to be in fact a boolean value.
```suggestion
      expect(rentalNFTComponentShouldFetch(filters)).toBeFalsy()
```

Reply:
It was already using Falsy